msu/systems/mod_settings/load.nut

```diff
@@ -122,4 +123,4 @@ logPage.add(verboseModeToggle);
 
 
 // Testing Code
-testSettingsSystem();
```

Comment:
This should be guarded by MSU.Mod.isEnabled("modsettings") instead

Reply:
Unsure what you mean here